### ISSUE TITLE
installation error

### ISSUE THREAD

**Issue by denfromufa** [quantopian/qgrid#16]
```
qgrid.nbinstall()  # copies javascript dependencies to your /nbextensions folder 
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-ca03556e1713> in <module>()
----> 1 qgrid.nbinstall()  # copies javascript dependencies to your /nbextensions folder

C:\Python\Python27\lib\site-packages\qgrid\__init__.pyc in nbinstall(user, overwrite)
     23         user=user,
     24         symlink=False,
---> 25         verbose=0,
     26     )

TypeError: install_nbextension() got an unexpected keyword argument 'user'

```

**Comment by dhirsch** [comment]
I have am running into the same issue. Fresh install of qgrid, using ipython 2.4.1 on Mac. Any ideas?

**Comment by ssanderson** [comment]
The `user` option is new in IPython 3.0, which is in release candidate now.  You can install it with `pip install IPython --pre`.  Once it's actually released, we'll probably make 3.0 a proper requirement, since there are a bunch of 3.0 features that benefit this project.
